### PR TITLE
updated-screenshots

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ X] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by stacykor — APPROVED** [review]

File: docs/send-data/sumo-distribution-opentelemetry.md [hunk 1]
```diff
@@ -18,6 +19,6 @@ The Collection page provides links to download the Sumo Logic Distribution for O
 Releases are available to download in the [Sumo Logic Distribution for OpenTelemetry](https://github.com/SumoLogic/sumologic-otel-collector/releases).
 :::
 
-1. Navigate to **Manage Data > Collection** and select the **Collection** tab. On the top right of the page, click **Add Collector**.<br/>![add-collector-link.png](/img/send-data/add-collector-link.png)
+1. Navigate to **Manage Data > Collection** and select the **Open Telemetry Collection** tab. On the top right of the page, click **Add Collector**.<br/>![add-collector-link.png](/img/send-data/add-collector-link.png)
```
Comment (kimsauce) [inline docs/send-data/sumo-distribution-opentelemetry.md:22]:
```suggestion
1. Navigate to **Manage Data > Collection** and select the **OpenTelemetry Collection** tab. On the top right of the page, click **Add Collector**.<br/>![add-collector-link.png](/img/send-data/add-collector-link.png)
```

**Review by kimsauce — APPROVED** [review]

File: docs/send-data/sumo-distribution-opentelemetry.md (hunk 1 above)
Comment (JainM6) [reply docs/send-data/sumo-distribution-opentelemetry.md]: @kimsauce Open Telemetry - two words as per the UI.

Comment (kimsauce) [reply docs/send-data/sumo-distribution-opentelemetry.md:22]: Oh I see in your screenshot above. I'll have the S3M fix it - this is an error and it appears as one word throughout the rest of the Docs. It's the official industry-wide spelling.

**Comment by kimsauce** [comment]
I've added the SMEs here as reviewers

**Pull request merged**